build-tools/scripts/BuildEverything.mk

```diff
@@ -164,8 +170,22 @@ package-oss $(ZIP_OUTPUT):
 		for f in `cat $$_sl` ; do \
 			echo "$(ZIP_OUTPUT_BASENAME)/bin/$$c/lib/xamarin.android/xbuild/$$f" >> "$$_exclude_list"; \
 		done; \
-	done; \
```

Comment:
This change will break the `zip -r` command, as without the continuation `$$_exclude_list` won't be defined, preventing `zip -r ... -x@$$_exclude_list` from working.

The `zip` will thus need to be updated to instead use `-x@.__exclude_list.txt`.